akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/HttpChallenge.scala

```diff
@@ -24,6 +25,8 @@ final case class HttpChallenge(scheme: String, realm: String,
 
 object HttpChallenges {
 
+  def negotiate(): HttpChallenge = HttpChallenge("Negotiate", "")
+
   def basic(realm: String): HttpChallenge = HttpChallenge("Basic", realm)
 
   def oAuth2(realm: String): HttpChallenge = HttpChallenge("Bearer", realm)
```

Comment:
Yeap. I don't think it's worth deprecating them to be honest.
They're far better here than in the Directives where we removed them.